api/core/domain/models/model_provider_datas_mapping.py

```diff
@@ -572,6 +572,11 @@
             completion_cost_per_token=0.000_015,
             source="https://aws.amazon.com/bedrock/pricing/",
         ),
+        lifecycle_data=LifecycleData(
+            sunset_date=datetime.date(year=2025, month=7, day=20),
+            source="https://aws.amazon.com/bedrock/pricing/",
+            post_sunset_replacement_model=Model.CLAUDE_3_5_SONNET_20241022,
```

Comment:
3.7 ?

Reply:
Ah we will see ? Maybe 3.5 for now since we have more rate limits